astronomy/лидов_古在_test.cpp

```diff
@@ -0,0 +1,165 @@
+using geometry::NonRotating;
+using geometry::Position;
+using integrators::SymmetricLinearMultistepIntegrator;
+using integrators::methods::Quinlan1999Order8A;
+using integrators::methods::QuinlanTremaine1990Order12;
+using physics::BodyCentredNonRotatingDynamicFrame;
+using physics::DiscreteTrajectory;
+using physics::Ephemeris;
+using physics::MassiveBody;
+using physics::MasslessBody;
+using physics::SolarSystem;
+using physics::Trajectory;
+using quantities::Cos;
+using quantities::Pow;
+using quantities::Sin;
+using quantities::si::Metre;
+using quantities::si::Milli;
+using quantities::si::Minute;
+using quantities::si::Radian;
+using quantities::si::Kilo;
+using quantities::si::Second;
+using testing_utilities::operator""_⑴;
+using quantities::si::Degree;
+using testing_utilities::IsNear;
+
+// A test that showcases the eccentricity-inclination exchange mechanism
+// described in [Лид61] and [Koz62].  We follow the treatment in [Лид61].
+class Лидов古在Test : public ::testing::Test {
+ protected:
+  using MercuryCentredInertial =
+      Frame<enum class MercuryCentredInertialTag, NonRotating>;
+
+  Лидов古在Test()
+      : solar_system_1950_(
+            SOLUTION_DIR / "astronomy" / "sol_gravity_model.proto.txt",
+            SOLUTION_DIR / "astronomy" /
+                "sol_initial_state_jd_2433282_500000000.proto.txt"),
+        ephemeris_(solar_system_1950_.MakeEphemeris(
+            /*accuracy_parameters=*/{/*fitting_tolerance=*/1 * Milli(Metre),
+                                     /*geopotential_tolerance=*/0x1p-24},
+            Ephemeris<ICRS>::FixedStepParameters(
+                SymmetricLinearMultistepIntegrator<QuinlanTremaine1990Order12,
+                                                   Position<ICRS>>(),
+                /*step=*/10 * Minute))),
+        mercury_(*solar_system_1950_.massive_body(*ephemeris_, "Mercury")),
+        mercury_frame_(ephemeris_.get(), &mercury_) {}
+
+  SolarSystem<ICRS> solar_system_1950_;
+  not_null<std::unique_ptr<Ephemeris<ICRS>>> ephemeris_;
+  MassiveBody const& mercury_;
+  BodyCentredNonRotatingDynamicFrame<ICRS, MercuryCentredInertial>
+      mercury_frame_;
+};
+
+#if !_DEBUG
+TEST_F(Лидов古在Test, MercuryOrbiter1) {
+  DiscreteTrajectory<ICRS> icrs_trajectory;
+  icrs_trajectory.Append(MercuryOrbiter1119InitialTime,
+                         MercuryOrbiter1119InitialDegreesOfFreedom<ICRS>);
+  icrs_trajectory.SetDownsampling(ksp_plugin::MaxDenseIntervals,
+                                  ksp_plugin::DownsamplingTolerance);
+  auto const instance = ephemeris_->NewInstance(
+      {&icrs_trajectory},
+      Ephemeris<ICRS>::NoIntrinsicAccelerations,
+      {SymmetricLinearMultistepIntegrator<Quinlan1999Order8A, Position<ICRS>>(),
+       /*step=*/10 * Second});
+
+  for (int year = 1967; year < 1980; ++year) {
+    Instant const t = DateTimeAsTT(date_time::DateTime::BeginningOfDay(
+        date_time::Date::Ordinal(year, 1)));
+    LOG(ERROR) << "Flowing to " << t;
```

Comment:
Not really an `ERROR`.  Should we log to `INFO` and make sure that `INFO` goes to `stderr`?  Same at line 96.